### DISCUSSION TITLE
Support Visual Studio 2022 and C#

### DISCUSSION THREAD

**Discussion by ZCY-G** [community/community#7715] [Copilot Conversations] [answered]
I hope Copilot can support Visual Studio 2022 and C#.

**Comment by FabienDarr** [comment]
How long have you been waiting to get it ?

**Comment by JamDoggie** [comment]
I second this. I've been using Copilot in Rider with C# and it's an incredibly amazing experience, but I much prefer Visual Studio over Rider. Would be very nice to get support, as it's been a very long time.

**Comment by kzu** [comment, marked as answer]
Wish granted! https://marketplace.visualstudio.com/items?itemName=GitHub.copilotvs

**Reply by JamDoggie** [reply]
Awesome!!! I opened vs a few days ago and saw copilot in the marketplace and was super hyped.
